tensorflow/python/kernel_tests/nn_ops/embedding_ops_test.py

```diff
@@ -645,9 +647,9 @@ def testRaggedMaxNorm(self):
     self.assertAllEqual(embedding, [[[1.0], [1.0]], [[1.0]]])
 
 
-class EmbeddingLookupSparseTest(test.TestCase):
+class EmbeddingLookupSparseTest(test.TestCase, parameterized.TestCase):
```

Comment:
Not really related to this PR. Could we add a comment to move this test to tensorflow/python/ops/embedding_ops_test.py? Thanks!